disnake/channel.py

```diff
@@ -365,6 +365,7 @@ async def edit(
         default_auto_archive_duration: Optional[AnyThreadArchiveDuration] = MISSING,
         type: ChannelType = MISSING,
         overwrites: Mapping[Union[Role, Member], PermissionOverwrite] = MISSING,
+        flags: ChannelFlags = MISSING,
```

Comment:
missing in overload

Reply:
https://github.com/DisnakeDev/disnake/pull/642/commits/615e105c3c93a8235c291b158991b1beb124196b